reana/reana_dev/run.py

```diff
@@ -476,12 +477,37 @@ def run_example(
 
     display_message(
         "Running the following matrix:\n"
-        "Demos: {}\n"
-        "Workflow engines: {}\n"
-        "Compute backends: {}".format(
-            ",".join(components), ",".join(workflow_engines), ",".join(compute_backends)
-        )
+        f"Demos: {','.join(components)}\n"
+        f"Workflow engines: {','.join(workflow_engines)}\n"
+        f"Compute backends: {','.join(compute_backends)}"
     )
+
+    def _verify_log_output(component: str, workflow_name: str) -> bool:
+        for log_message in get_expected_log_messages_for_example(component):
+            cmd = f"reana-client logs -w {workflow_name} | {{ grep -c '{log_message}' || true; }}"
```

Comment:
`|| true` may look weird there... If we don't want to use the exit status of the command at this place, we can use `grep text | wc -l` perhaps. This is classic, so may be more readable. (It also won't exit whether there is 0 match or more matches.)